src/main/resources/gui/index.html

```diff
@@ -57,11 +59,29 @@
 
     fetch('/', requestOptions)
       .then(response => response.text())
-      .then(result => output.innerText = result)
+      .then(result => {
+        output.innerHTML = result;
+        report = result;
+        download.disabled = false;
```

Comment:
This should be disabled prior request then, or not?

Reply:
Actually "download.disabled = false" activates the button. The button is disabled at start. See in README.md Line 16

<input type="button" id="dwn-btn" value="Download Report" onclick="return reporting();" **disabled**>

Reply:
I had a second submit in mind, with a long processing time. Is your implementation safe then? (not double checked it)

Reply:
I implemented a check to the javascript that should prevent downloading if there is not data.

Reply:
Sorry for not reviewing for long time. I wanted to prepare a release and put this into it. I checked your implementation and noticed just now that only focus on included html. I guess your are - as most of the users - using the validator with xrechnung configuration and then it makes sense to somehow extract the included html fragments.

But including html fragments into the report is a decission xrechnung validator configuration project did. The validator does not know about "embedded" reports/html. And since we have other configurations as well, extracting/downloading only parts of the report wouldn't make sense. Only downloading the complete report would.